packages/mobile-top-nav/src/assets/img/ia-icon.js

```diff
@@ -0,0 +1,35 @@
+import { LitElement, html } from 'lit-element';
+import './icons';
+
+class IAIcon extends LitElement {
```

Comment:
I think `ia-icon` should be pulled out to be its own module/package. We're going to be using this across many other contexts so it doesn't really make sense to live in here.

Reply:
yah, +1.  I remember us mentioning this and agreeing as whole and also agreeing that we can do this as an after effect of getting the mobile nav merged.